packages/move/sources/vesting.move

```diff
@@ -109,6 +135,14 @@ module deployment_addr::vesting {
         config.pending_admin = option::none();
     }
 
+    //// - - - CLAIMER FUNCTIONS - - -
+
+    // FIX: given the above suggested fixes, the VestingStream would have to track the claimer for that stream
+    // the claim_tokens function would have to be adjusted to accept Object<VestingStream>
+    // Additionally, it would not have to accept sender: &signer, since anyone should be able to call claim_tokens
+    // which should transfer all tokens available to claim to the claimer
+    // Additionally, on a claim_tokens call that is beyond the end time, you could cleanup storage and delete the VestingStream
```

Comment:
I was also thinking about the cleanup, but decided against it since a frontend might want to show vesting streams that have already ended.

Reply:
frontends would typically rely on indexing services that we'd develop say a sentio aptos processor that implements event handlers that handle events emitted by the module.

frontends would typically call view functions only as a fallback if the indexing service is unavailable for whatever reason.